src/base/utils.ml

```diff
@@ -1,5 +1,4 @@
 open Core_kernel
```

Comment:
I removed `utils.mli` because it's a pain to handle the interface when refactoring (and this is all going to disappear anyway)